File: sdks/java/src/main/java/com/boltprediction/sdk/model/TenantCreateRequest.java
package com.boltprediction.sdk.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import java.util.List;
import java.util.Map;
import java.time.OffsetDateTime;

/** TenantCreateRequest */
@JsonIgnoreProperties(ignoreUnknown = true)
public class TenantCreateRequest {

    @JsonProperty("tenant_code")
    private String tenantCode;

    @JsonProperty("tenant_name")
    private String tenantName;

    @JsonProperty("contact_email")
    private Object contactEmail;

    @JsonProperty("contact_phone")
    private Object contactPhone;

    @JsonProperty("expire_time")
    private Object expireTime;

    public String getTenantCode() {
        return tenantCode;
    }

    public void setTenantCode(String tenantCode) {
        this.tenantCode = tenantCode;
    }

    public String getTenantName() {
        return tenantName;
    }

    public void setTenantName(String tenantName) {
        this.tenantName = tenantName;
    }

    public Object getContactEmail() {
        return contactEmail;
    }

    public void setContactEmail(Object contactEmail) {
        this.contactEmail = contactEmail;
    }

    public Object getContactPhone() {
        return contactPhone;
    }

    public void setContactPhone(Object contactPhone) {
        this.contactPhone = contactPhone;
    }

    public Object getExpireTime() {
        return expireTime;
    }

    public void setExpireTime(Object expireTime) {
        this.expireTime = expireTime;
    }

}

File: sdks/python/bolt_prediction_sdk/models/tenant_create_request.py
"""
TenantCreateRequest 模型
"""

from typing import Optional, Any, Dict, List
from datetime import datetime
from pydantic import Field

from .base import SDKBaseModel


class TenantCreateRequest(SDKBaseModel):
    """TenantCreateRequest"""

    tenant_code: str = Field(description="租户编码")
    tenant_name: str = Field(description="租户名称")
    contact_email: Optional[Any] = Field(description="联系邮箱", default=None)
    contact_phone: Optional[Any] = Field(description="联系电话", default=None)
    expire_time: Optional[Any] = Field(description="到期时间", default=None)
